mlflow/tracking/fluent.py

```diff
@@ -357,10 +570,37 @@ def create_experiment(name, artifact_location=None):
     """
     Create an experiment.
 
-    :param name: The experiment name. Must be unique.
+    :param name: The experiment name, which must be unique and is case sensitive
     :param artifact_location: The location to store run artifacts.
                               If not provided, the server picks an appropriate default.
     :return: Integer ID of the created experiment.
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+
+        # Create an experiment name, which must be unique and case sensitve
+        experiment_id = mlflow.create_experiment("Social NLP Experiments")
+
+        # Convert an experiment ID as a string argument and fetch its data
+        experiment = mlflow.get_experiment(str(experiment_id))
```

Comment:
```suggestion
        experiment = mlflow.get_experiment(experiment_id)
```

We don't need to use `str` here because `experiment_id` is alread a string:

```python
import mlflow

print(type(mlflow.create_experiment("foo")))  # -> class 'str'
```

Reply:
The docstring [return argument for create_experiment ](https://mlflow.org/docs/latest/python_api/mlflow.html#mlflow.create_experiment)is incorrect.

Reply:
done!